### DISCUSSION TITLE
Query in PostgreSqlSchemaHelper.getListTablesSQL does not work as intended

### DISCUSSION THREAD

**Discussion by pphunor72** [mikro-orm/mikro-orm#2180] [Q&A]
It filters out schemas beginning with 'pg', despite of 'pg_' filter condition. This happens because the underscore character should be escaped in postgresql, otherwise it matches any single character. See https://www.postgresql.org/docs/current/functions-matching.html

**Comment by B4nan** [comment]
Please dont ignore issue templates.
